sql/telemetry/core_live/view.sql

```diff
@@ -0,0 +1,27 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.core_live`
```

Comment:
I think this would be better suited to living in the `telemetry_live` dataset, and I'm also interested in @jklukas opinion on the matter:
```suggestion
  `moz-fx-data-shared-prod.telemetry_live.core`
```

Reply:
Views on top of a live table are an edge case that we haven't fully grappled with. I can see an argument for putting this in `telemetry_live`, but it's forbidden by the permissions (Airflow should not have access to create tables/views in the `_live` datasets) and logically I want to keep `_live` and `_stable` as being purely owned by the pipeline.

So I think `moz-fx-data-shared-prod.telemetry.core_live` as used here seems like a reasonable option.

Reply:
Although I also might argue we should put this in `telemetry_derived` to hide it as an implementation detail rather than making it user-facing.

Reply:
good point, i'm on board with putting this in `telemetry_derived`:
```suggestion
  `moz-fx-data-shared-prod.telemetry_derived.core_live`
```

Reply:
I don't have any strong feelings on where this should be.  `telemetry.core_live` is nice for consistency with `telemetry.core` and the other views but I understand wanting to hide it.  @relud Final decision?

Reply:
> @relud Final decision?

use `telemetry_derived`, if people outside data eng are reading live pings then the implied contract is `ymmv, but we'll try to help`, and not `this is supported`